src/twisted/internet/test/test_unix.py

```diff
@@ -17,9 +17,11 @@
 from unittest import skipIf
 
 try:
-    from socket import AF_UNIX
+    from socket import AF_UNIX as _AF_UNIX
 except ImportError:
     AF_UNIX = None
+else:
+    AF_UNIX = _AF_UNIX
```

Comment:
I would suggest ` # type: ignore[assignment]` on the line where we assign `None` instead.

Reply:
@wsanchez I implemented this pattern:

```
try:
      from blah import foo as _foo
except ImportError
     foo = None
else:
     foo = _foo
```

Based on the discussion in this mypy issue:
https://github.com/python/mypy/issues/1297#issuecomment-508593494

I tried to avoid using mypy comments as much as possible, and go with restructuring the Python code, unless changing the code made things worse/complicated.

Instead of following that approach, are you recommending that it is better to go with the mypy comment so that:

```
try:
    from blah import foo
except ImportError:
    foo = None  # type: ignore[assignment]
```

??